src/Chainweb/Chainweb.hs

```diff
@@ -455,7 +455,7 @@ withChainwebInternal conf logger peer rocksDb dbDir nodeid resetDb inner = do
 
     synchronizePactDb cs cutDb = do
         currentCut <- _cut cutDb
-        mapM_ syncOne $ mergeCutResources $ _cutMap currentCut
+        mapConcurrently_ syncOne $ mergeCutResources $ _cutMap currentCut
```

Comment:
This will use all available threads. Is that desired?

Reply:
For 20 chains during initialization, yes. It should be almost a no-op in most cases (we are investigating currently why it isn't currently). Also, if at all it takes longer it should be IO bound.

Reply:
But once we go to, say >100, chains, we should use a scheduler.

Reply:
@fosskers if you are familiar with the scheduler package and know how to fix this quickly, please do it.